tests/net/lib/dns_addremove/src/main.c

```diff
@@ -0,0 +1,572 @@
+/*
+ * Copyright (c) 2019 Laird Connectivity

```

Comment:
As most of the code is copied from other tests, I would prefer that the original copyright is also kept here. You can add your copyright too, no problem with that.

Reply:
Only the interface setup comes from another test, the rest of the code is not copied over.

Reply:
Ok, then it is ok. For small functions the copyright is not that important.